spec/factories/user.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :user do
+
+  end
+end
```

Comment:
don't leave empty and not needed files, and if they will be needed in a separate task - then the appropriate dev will create them in his task